intro-HPC/ch_debug_gent.tex

```diff
@@ -0,0 +1,46 @@
+\chapter{HPC-UGent debug cluster}
+\label{ch:debug_ugent}
+
+
+\section{Purpose}
+\label{sec:debug_ugent_pupose}
+The purpose of the debug cluster is to run debug jobs. Additionally, it can be also used for courses. 
```

Comment:
"The purpose of this cluster is to give the user an environment where there should be no waiting in the queue to get access to a limited number of resources. This environment allows a user to immediatelty start doing, and is the ideal place for interactive work such as development, debugging and light prodcution (typically sufficient for training and/or courses). This enviroment should be seen as an extension of the loginnodes, instead of the a compute resource."

some remarks:

- i would really stress the interactive part more than the debug.
- we should also recommend it for OOD/portal